difflint/lib.py

```diff
@@ -249,13 +197,31 @@ def detect_new_diff_lint_errors(log_output):
 
 
 def main():
-    if not lint_script_exists() or not all_files_configured():
-        return
+    parser = argparse.ArgumentParser(description='Linter that will examine ' +
+        'only new changes as you commit them.')
```

Comment:
nit: Vertical alignment. (The way I usually deal with the problem of everything ending up way indented is by breaking after the assignment operator in a case like this.)